replibyte/src/source/mysql.rs

```diff
@@ -81,7 +81,6 @@ impl<'a> Source for Mysql<'a> {
             "--complete-insert",   // have column names in INSERT INTO rows
             "--single-transaction", // https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction
             "--quick", // reads out large tables in a way that doesn't require having enough RAM to fit the full table in memory
-            "--databases",
```

Comment:
@karpa4o4 I am not sure if it is wise to remove this parameter - WDYT @fabriceclementz ?

Reply:
This parameter is needed to enumerate the list of databases, but because only one is indicated here - it is redundant.
https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_databases